### PR TITLE
catkinize ar_tools (fixes #2)

### LINKED ISSUE #2: Catkin Support
n/t

### PR DESCRIPTION
**Pull request by xqms**
Here is a catkinized version of `ar_tools` (see #2).

`ARToolKit` is downloaded using the ExternalProject CMake module, analog to the `download_unpack_build` capability `rosmake`.

I added myself as maintainer in the `package.xml` files, since one maintainer is mandatory and I couldn't find your email. Please change that.

### CODE REVIEW

**Comment by dlaz** [comment]
@LucidOne is this ever going to get merged? If you don't have time, it might make sense for someone else to take over maintainership (@xqms?)

**Comment by xqms** [comment]
@dlaz: I'm not really interested in maintaining this. I'm not even using `ar_tools` anymore, so probably someone else would be better suited.

**Comment by dlaz** [comment]
I would be willing to take over maintainership. @LucidOne, if that's ok with you, can you transfer ownership of the repo to me?

File: artoolkit/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+)
+
+include(ExternalProject)
+ExternalProject_Add(ARToolkit
+	# Settings:
+	#  out of source build is not supported
+	BINARY_DIR ARToolkit-prefix/src/ARToolkit
+
+	# Download
+	URL http://csc.kth.se/~fevb/downloads/ARToolKit-SVN-r311.tar.gz
```
Comment (LucidOne) [inline artoolkit/CMakeLists.txt]: A long-term plan for this needs to be resolved in the future.

**Comment by LucidOne** [comment]
I also want to apologize for the delays.
Co-maintainers have been added so this won't happen again.

I will also endeavor to avoid letting perfection stop progress.
One day I hope that I won't have a day job that gets in the way of writing code.

Thanks for the contributions!

**Pull request merged**